service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingVenueLookUpService.java

```diff
@@ -29,6 +31,13 @@ public HearingVenueLookUpService(ObjectMapper objectMapper) {
         populateHearingVenueMappings();
     }
 
+    public HearingDateDynamicElement getHearingDynamicElement(DynamicList list) {
```

Comment:
would it be worth moving this into _HearingBookingService_ as its more related to hearing booking than to hearing venue?

Reply:
Yeah you're right, will move it over